components/salesforce_rest_api/actions/sosl-search/sosl-search.mjs

```diff
@@ -0,0 +1,29 @@
+import salesforce from "../../salesforce_rest_api.app.mjs";
+import { toSingleLineString } from "../../common/utils.mjs";
+
+export default {
+  key: "salesforce_rest_api-sosl-search",
+  name: "SOSL Search",
+  description: toSingleLineString(`
+    Executes the specified SOSL search.
+    See [docs](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_sosl.htm)
+  `),
+  version: "0.2.2",
+  type: "action",
+  props: {
+    salesforce,
+    search: {
+      type: "string",
+      label: "SOSL Query",
+      description: "A SOSL search query",
+    },
+  },
+  async run({ $ }) {
+    const response = await this.salesforce.search({
+      $,
+      search: this.search,
+    });
+    $.export("$summary", "Returned results for SOSL search");
```

Comment:
Do we have the quantity of returned results? Maybe we can say
```suggestion
    $.export("$summary", "Successfully returned ${response.length} results for SOSL search");
```
If we found no records so we could say

```suggestion
    $.export("$summary", "No results found for SOSL search");
```